### PR TITLE
0162. Find Peak Element

### PR DESCRIPTION
**Pull request by Rittika1234**

### CODE REVIEW

**Comment by wingkwong** [comment]
Next time please don't close and open a new PR. you can just push the changes to your branch and the changes will be reflected in the PR automatically.

**Comment by Rittika1234** [comment]
Okay Sorry. I will not do this next time.  Please check my PR.

**Review by wingkwong — CHANGES_REQUESTED** [review]
Do you think this is ready for review? Please look at other pages and verify it locally first.

image

**Comment by Rittika1234** [comment]
Check my PR

File: solutions/0100-0199/0162.Find Peak Element.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+</Tabs>
+
+
+## Approach 2: Binary Search
+
+See [Binary Search](../../tutorials/basic-topics/binary-search) for basic understanding.
+
+
+Time complexity: $O(log(n))$
+Space complexity:  $O(1)$
```
Comment (wingkwong) [inline solutions/0100-0199/0162.Find Peak Element.md:82]: these will be rendered in one line in the site which results in poor readability. I highly suggest you to start the app locally and verify first.

File: solutions/0100-0199/0162.Find Peak Element.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+* 1 <= nums.length <= 1000
+* -231 <= nums[i] <= 231 - 1
+* nums[i] != nums[i + 1] for all valid i.
+
+## Approach 1: Brute Force
+
+This is a brute force approach. We traverse each element and check whether it is greater than to max value.If we find one, than we initialize this value to max, return max value.
+
+Time complexity: $O(n)$
+Space complexity: $O(1)$
```
Comment (wingkwong) [inline solutions/0100-0199/0162.Find Peak Element.md:49]: these will be rendered in one line in the site which results in poor readability. I highly suggest you to start the app locally and verify first.

File: solutions/0100-0199/0162.Find Peak Element.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            } else {
+                
+                start = mid + 1; 
+            }
+        }
+      
+        return start; 
+    }
+}
+```
```
Comment (wingkwong) [inline solutions/0100-0199/0162.Find Peak Element.md:110]: where is the explanation ?? what is the intuition of `arr[mid] > arr[mid+1]` ?

File: solutions/0100-0199/0162.Find Peak Element.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+**Constraints:**
+
+* 1 <= nums.length <= 1000
+* -231 <= nums[i] <= 231 - 1
+* nums[i] != nums[i + 1] for all valid i.
+
+## Approach 1: Brute Force
+
+This is a brute force approach. We traverse each element and check whether it is greater than to max value.If we find one, than we initialize this value to max, return max value.
```
Comment (wingkwong) [inline solutions/0100-0199/0162.Find Peak Element.md:46]: The statement `If we find one, than we initialize this value to max, return max value.` is not accurate and doesn't match with your code.  $max$ is updated as long as it meets a larger number, not just "initialize" and  return.

**Review by wingkwong — CHANGES_REQUESTED** [review]

**Pull request closed without merging**